numba_dpex/tests/debugging/test_info.py

```diff
@@ -19,64 +19,67 @@
 pytestmark = skip_no_gdb
 
 
-def side_by_side_case(api):
+def side_by_side_info_case(api):
+    if api == "numba":
+        env = {"NUMBA_OPT": 0}
+    else:
+        env = {"NUMBA_DPEX_OPT": 0}
+
     return (
+        env,
         side_by_side_breakpoint,
         f"side-by-side.py --api={api}",
         None,
         (
-            r"param_a = 0",
-            r"param_b = 0",
+            r"param_a = .*",
+            r"param_b = .*",
         ),
+    )
+
+
+def side_by_side_print_case(api):
+    if api == "numba":
+        env = {"NUMBA_OPT": 0}
+    else:
+        env = {"NUMBA_DPEX_OPT": 0}
+
+    return (
+        env,
+        side_by_side_breakpoint,
+        f"side-by-side.py --api={api}",
+        None,
         (
             "param_a",
-            r"\$1 = 0",
+            r"\$1 = .*",
             r"type = float32",
             r"type = float32",
         ),
     )
 
 
+@pytest.mark.skip  # https://github.com/IntelPython/numba-dpex/issues/1232
```

Comment:
Can we mark it as `xfail` ? Does it affect execution time? (Same for other places if they exists)

Reply:
It may pass sometimes which will result into failure due to xfail. https://github.com/IntelPython/numba-dpex/issues/1232